doc/rados/operations/devices.rst

```diff
@@ -33,103 +37,103 @@ location and how it is being consumed with:
 Identifying physical devices
 ----------------------------
 
-You can blink the drive LEDs on hardware enclosures to make the replacement of
-failed disks easy and less error-prone.  Use the following command::
+To make the replacement of failed disks easier and less error-prone, you can
```

Comment:
Suggest `blink the drive's locator LED by`

There are situations where this will and won't work (kernel revision, SES firmware, stupid HBA tricks), tempted to suggest something like s/can/may be able to/

Reply:
I'll beef this part up with an explanation that should give the first-time reader a good sense of this territory.